scripts/test/lint.py

```diff
@@ -272,6 +273,36 @@ def check(self, path):
         folder = path.parts[-2]
 
         return not (pkg_id == folder == nuspec[:-len(".nuspec")])
+        
+        
+        
+class UsesInvalidCategoryNuspec(Lint):
```

Comment:
I think the linter that checks the category is one in `categories.txt` should be named `UsesInvalidCategory` and the one that checks that the installer/uninstaller use the category from the nuspec be named something like `UsesCategoryFromNuspec`